app/Nova/Actions/ExportUsersBuzzCardAccess.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Actions;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\URL;
+use Laravel\Nova\Actions\Action;
+use Laravel\Nova\Fields\ActionFields;
+use Laravel\Nova\Fields\Select;
+
+class ExportUsersBuzzCardAccess extends Action
+{
+    /**
+     * The displayable name of the action.
+     *
+     * @var string
+     */
+    public $name = 'Export BuzzCard Access List';
+
+    /**
+     * Disables action log events for this action.
+     *
+     * @var bool
+     */
+    public $withoutActionEvents = true;
+
+    /**
+     * Perform the action on the given models.
+     *
+     * @param \Laravel\Nova\Fields\ActionFields  $fields
+     * @param \Illuminate\Support\Collection  $models
+     *
+     * @return array<string,string>
+     */
+    public function handle(ActionFields $fields, Collection $models): array
+    {
+        $population = $fields->population;
+        $query = User::query();
```

Comment:
My point about using `when` is that you can write a single query that has the shared statements and the `when` all chained together. You can use the `when` as an if/else.

As an example:
https://github.com/RoboJackets/apiary/blob/b1f88d60c5e6e4dd3729f708f1b0792dbaa28a35/app/Nova/Metrics/TotalCollections.php#L25-L44